recipes/pelegant/meta.yaml

```diff
@@ -0,0 +1,275 @@
+{% set name = "pelegant" %}
+{% set version = "2023.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://ops.aps.anl.gov/downloads/elegant.{{ version }}.tar.gz
+    sha256: 48efea2f34d055807f7ae6b31c1f115ab75730743c1c238873fa9c83e8f48085
+    folder: src/elegant
+  - url: https://ops.aps.anl.gov/downloads/oag.apps.configure.tar.gz
+    sha256: eef0912a70b069e55d1df24cd008f42e6a5923e35461f2b69f92cd1e7e2d4596
+    folder: src/oag-apps
+  - url: https://ops.aps.anl.gov/downloads/SDDS.5.5.tar.gz
+    sha256: c07823e10ba75bea4d6277a2cddf126f366e4ea563fea3967dd8268c7a13be2a
+    folder: src/sdds
+  - url: https://ops.aps.anl.gov/downloads/epics.base.configure.tar.gz
+    sha256: d18a5a1dd58ab485df32a55a32f0f2a7517b6a5361439208dc38d2fe69f6e22e
+    folder: src/epics-base
+  - url: https://ops.aps.anl.gov/downloads/epics.extensions.configure.tar.gz
+    sha256: 46db83bf7e0cc70be663a822e4cb229cb03f52e98e6920fbc1ce181df9eac59b
+    folder: src/epics-extensions
+
+build:
+  skip: true  # [win]
+  skip: true  # [py<310]
+  number: 0
+  ignore_run_exports:
+    # libsz/libaec used in hdf2sdds/sdds2hdf:
+    - libaec
+    - python
+    # Darwin-only; linked in sdds2tiff/tiff2sdds
+    - zstd  # [osx]
+    - libgomp  # [linux]
+    - libopenblas  # [linux]
+    - liblapack  # [linux]
+    - libblas  # [linux]
+
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("fortran") }}
+    - mpich
+    - make
+  host:
+    - blas * *openblas
+    - libopenblas
+    - libblas * *openblas  # [linux]
+    - liblapack * *openblas  # [linux]
+    - fftw * mpi_mpich_*
+    - gsl
+    - hdf5 * mpi_mpich_*
+    - libaec
+    - libgomp  # [linux]
+    - mpi * mpich*
+    - mpich
+    - ncurses
+    - openmp  # [linux]
+    - python
+    - xz
+    - zlib
+    - zstd  # [osx]
+  run:
+    - blas * *openblas
+    - libopenblas
+    # Note: Listed specifically for the conda-verify linkage checker; this is
+    # already included in the metapackage 'blas':
+    - liblapack * *openblas  # [linux]
+    - libblas * *openblas  # [linux]
+    - fftw * mpi_mpich_*
+    - gsl
+    - hdf5 * mpi_mpich_*
+    - libgomp  # [linux]
+    - mpi * mpich*
+    - mpich
+    - ncurses
+    - openmp  # [linux]
+    - python
+    - xz
```

Comment:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#export-runtime-requirements

Reply:
"run_exports" is a statement in a recipes which causes downstream recipes to add dependencies to their run requirements. e.g. If the recipe for "foo" depends on "libpng" at build time, "libpng" exports a run dependency on "libpng" into the "foo" recipe. The run dependency of "foo" on "libpng" is needed because "libpng" is a dynamically linked library so it is needed at runtime. Run exports exist because the packagers of "libpng" should be more familiar with ABI compatibility than downstream packages.

Thus, it shouldn't be necessary to list many of these packages in the run requirements of elegant because the appropriate run requirements should already be exported from the upstream packages.